src/emcpy/plots/create_plots.py

```diff
@@ -1,14 +1,150 @@
 import numpy as np
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
 import matplotlib.pyplot as plt
+from matplotlib import rcParams
 from matplotlib.colors import Normalize
 from scipy.interpolate import interpn
+from cartopy.mpl.ticker import LongitudeFormatter, LatitudeFormatter
+from emcpy.plots.map_tools import Domain, MapProjection
 from emcpy.stats import get_linear_regression
 
+__all__ = ['CreatePlot', 'CreateMap']
 
-__all__ = ['CreatePlot']
 
+class EMCPyPlots:
 
-class CreatePlot:
+    def __init__(self):
```

Comment:
I like this, but I think this default constructor should return a figure handle via `fig=plt.figure(figsize=(12,8))`. One could provide a `figsize` as input argument if necessary/